source/ui/Components/Output/PixelDisplay.qml

```diff
@@ -0,0 +1,221 @@
+            }
+        }
+        Row {
+            anchors.fill: parent
+            anchors.leftMargin: 15
+            anchors.topMargin: 15
+            anchors.rightMargin: 15
+            anchors.bottomMargin: 15
+
+            spacing: 15
+
+            // Title of each settings control.
+            Column {
+                spacing: 16
+                Text {
+                    text: "Pixel Base Address:"
+                }
+                Text {
+                    text: "Color Base Address:"
+                }
+                Text {
+                    text: "Color Mode:"
+                }
+                Text {
+                    text: "width:"
+                }
+                Text {
+                    text: "height:"
+                }
+            }
+            Column {
+                id: controlsColumn
+                spacing: 8
+
+                function integerFromInputString(input) {
+                    var base = 10;
+                    if (input.indexOf("0x") == 0) {
+                        base = 16;
+                        input = input.slice(2);
+                    } else if (input.indexOf("0b") == 0) {
+                        input = input.slice(2);
+                        base = 2;
+                    }
+                    return parseInt(input, base);
+                }
+
+                TextField {
+                    id: pixelBaseAddressTextField
+
+                    onAccepted: { processInput(); }
+                    onEditingFinished: { processInput(); }
+
+                    function processInput() {
+                        var inputValue = controlsColumn.integerFromInputString(String(pixelBaseAddressTextField.text))
+                        if (inputValue >= 0) {
+                            pixeldisplaypainteditemid.pixelBaseAddress = inputValue
+                        }
+                    }
+                }
+
+                TextField {
+                    id: colorBaseAddressTextField
+
+                    onAccepted: { processInput(); }
+                    onEditingFinished: { processInput(); }
+
+                    function processInput() {
+                        var inputValue = controlsColumn.integerFromInputString(String(colorBaseAddressTextField.text))
+                        if (inputValue >= 0) {
+                            pixeldisplaypainteditemid.colorBaseAddress = inputValue
+                        }
+                    }
+                }
+
+                ComboBox {
+                    id: colorModeComboBox
+                    //width: 200
+                    model: [ "Monochrome", "RGB" ]
+                    onAccepted: {
+                        pixeldisplaypainteditemid.colorMode = colorModeComboBox.currentIndex
+                    }
+                    onActivated: {
+                        pixeldisplaypainteditemid.colorMode = colorModeComboBox.currentIndex
+                    }
+                }
+
+                // TextField {
```

Comment:
Please go through all these comments and see if you need any of the code. If not, remove them.